docs/3.0.0-beta.x/community/contribute-with-docs.md

```diff
@@ -0,0 +1,486 @@
+
+### Reference topics
+
+**Reference topics** are subjects that **are resources** to bookmark and used as a reference. Developers can be expected to refer to Reference topics regularly throughout the development of Strapi based projects.
+
+This _audience member_ is a developer who is already developing projects with Strapi. **Reference topics** are meant to be used repetitively as an up-to-date and managed resource for recipes, best practices, and sample code which can be copy/pasted.
+
+Examples include the [Models Guide](/3.0.0-beta.x/guides/models.html#how-to-create-a-model) and the [Controllers Guide](/3.0.0-beta.x/guides/controllers.html).
+
+### Explanations
+
+**Explanations** cover why something was done and provide an understanding of the choices made through the course of the development of Strapi. These are generally written as **Articles** and found in the [Strapi Blog](https://blog.strapi.io).
+
+This _audience member_ is a developer who is already developing projects with Strapi or who is studying whether Strapi is a good fit for their project(s). **Explanations** are meant to explain different choices in the Roadmap, overall direction, and how certain features generally work.
+
+Examples include [Why we split the management of the Admin user and the Front-end Users](https://blog.strapi.io/why-we-split-the-management-of-the-admin-users-and-end-users/) and [Restricting Permissions in Strapi Admin](https://blog.strapi.io/admin-permissions/).
+
+These four categories follow the recommendations found in this article about [documentation by Divio](https://www.divio.com/blog/documentation/). We recommend reading it to gain a complete understanding of each type of content contained in our documentation.
+
+## The Workflow
+
+Please read the following sections before moving to create documentation. (Either new documentation or modifying existing documentation.)
+
+1. [Technical Requirements and the Installation of the Docs](/3.0.0-beta.x/community/contribute-with-docs.html#technical-requirements)
+2. [Understanding the Docs Structure](/3.0.0-beta.x/community/contribute-with-docs.html#understanding-the-docs-structure)
+3. [Submitting new documentation](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-totally-new-documentation)
+4. [Adding/modifying/deleting documentation](/3.0.0-beta.x/community/contribute-with-docs.html#adding-modifying-deleting-content)
+
+- How to [submit an externally hosted tutorial](/3.0.0-beta.x/community/contribute-with-docs.html#submitting-an-externally-hosted-tutorial).
+- How to use the [Docs Style Guide](/3.0.0-beta.x/community/contribute-with-docs.html#docs-style-guide) to write better Docs.
+
+### Technical Requirements
+
+1. You have [Node v10.x.x](https://nodejs.org/en/download/) (LTS) or [Node v12.x.x](https://nodejs.org/en/download/current/) and [Yarn v1.2.0+](https://yarnpkg.com/en/).
```

Comment:
Well I think it should be good to create a specific file that explain the documentation setup so you don't have to repeat the steps in both contribute with code and contribute with docs parts.

Reply:
I agree. Maybe this can be done after this is validated and merged. I personally still have unresolved questions on how to accomplish this.

These includes will take some thought and I think they should be their own action on the docs.